### DISCUSSION TITLE
setting yScale min value for XYchart

### DISCUSSION THREAD

**Discussion by davidhan527** [airbnb/visx#1242] [Q&A] [answered]
I'm in the middle of converting a nivo line chart to use the visx xychart and I'm getting stuck on this one part. 

Nivo allows you to pass this to the yScale prop

```
        yScale={{
          type: "linear",
          min: "auto",
          max: "auto",
        }}
```

Setting min to "auto" will automatically find your minimum y value and set it to that instead of 0. This will work even though there are multiple lines in the line chart and it will find the minimum y value for all the lines.

 Is there a way to do the same thing using XYchart when there are multiple lines for a line chart?

**Comment by williaster** [comment, marked as answer]
Hey @davidhan527 👋  thanks for checking out `visx`. This question is pretty similar to the discussion in #1147, so maybe check that out.

Overall the x/y-scale config for `@visx/xychart` mirrors what's available in the `@visx/scale` package so the [scale docs](https://airbnb.io/visx/docs/scale) might be useful. By default `linear` type scales in `XYChart` will automatically set the min/max based on the data from all series but they **will also include zero** (mostly since this is often data vis best practice, so we default to that). You can turn this off simply by setting `yScale={{ type: 'linear', zero: false }}`. 

Again, there's more info in the other discussion about controlling the range, but hopefully this helps with your immediate problem!

**Reply by davidhan527** [reply]
@williaster Thank you for the quick response!
